frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -20,6 +20,8 @@ export enum ViewType {
     PATHS = 'PATHS',
 }
 
+export const TRENDS_BASED_INSIGHTS = ['TRENDS', 'STICKINESS', 'LIFECYCLE'] // Insights that are based on the same `Trends` components
```

Comment:
We don't use `ViewType` here due to some clash between types (see #4250 for details)